source/sass/global.scss

```diff
@@ -38,6 +38,21 @@ hr {
     width: 100%;
     overflow: hidden;
 
+    &.homepage {
+      $bg-pattern-height: 400px;
```

Comment:
@mmmavis this is really more of an SCSS stub than anything, to get a rough idea before we actually write the real SCSS =)